server/app/src/data_harvester.py

```diff
@@ -259,6 +210,58 @@ def copy_snapshots(self, datasets: List[StandardizedDataset]):
         # Delete the tmp folder
         os.system('rm -rf ' + tmp_folder)
 
+    @staticmethod
+    def id_short_trials(tmp_folder: str):
```

Comment:
It'd be good to have a type signature on the return value from this method, something like:
`def id_short_trials(tmp_folder: str) -> List[str]:`